### ISSUE TITLE
Use Mixins and allow option for no Initializers

### ISSUE THREAD

**Issue by jmurphyau** [sir-dunxalot/ember-tooltips#2]
Hi

I like this library but would prefer that it didn't load code on every single one of my components and views. The code in the 'didInsertElement' is run for every element inserted in my app - all the time. This shouldn't be the default behaviour - or it should at least be able to be turned off.

Also - I see that you've actually got the code that is inserted on every view/component in the initializer - it would be better if this was a Mixin that could be imported separately.. Allowing the initializer to be turned off and providing a mixin would allow people to put the tooltip only on components they want..

If you are open to these suggestions I may spend time implementing them and doing a PR - let me know your thoughts.

**Comment by sir-dunxalot** [comment]
Definitely valid points. I think there's some value in the 'plug and play' nature of initializers so perhaps an option, as you suggested, to manually import the code that's refactored into a mixin would be the best option. I'd happily accept a PR as I won't be working much until next week but either way would like to make these improvements.

The option is probably best set in the `config/environment` file because it's not really affecting the broccoli build, just the browser code. So something like:

```js
/* jshint node: true */

module.exports = function(environment) {
  var ENV = {
    modulePrefix: 'dummy',
    podModulePrefix: 'dummy/resources',
    environment: environment,
    baseURL: '/',
    locationType: 'auto',
    EmberENV: {
      FEATURES: {
        // Here you can enable experimental features on an ember canary build
        // e.g. 'with-controller': true
      }
    },

    APP: {
      // Here you can pass flags/options to your application instance
      // when it is created
    },

    tooltips: {
      setupInInitializer: true, // or false
    }
  };
}
```

**Comment by sir-dunxalot** [comment]
This will be fixed by #3, which is currently up for review.

**Comment by sir-dunxalot** [comment]
#3 merged into master and released as `0.2.0`.
